### PR TITLE
OwnerAddress page, ssv-api service

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

File: app/backend/services/ssv-api/ssvApiService.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  }
+
+  async getAccountData(address: string) {
+    return await this.ssvApi.request(METHOD.GET, `/accounts/${address}`);
+  }
+
+  async getOperatorsByIds(ids: string[] | number[]) {
+    return await this.ssvApi.request(METHOD.POST, '/operators', {ids});
+  }
+}
```
Comment (meshin-dev) [inline app/backend/services/ssv-api/ssvApiService.ts:19]: You can combine both ssv-api.ts and this file together. I'ts ok that SSVApi class would have extra-methods. Not necessary to have two classes

File: app/backend/services/ssv-migration/ssv-migration.service.ts [hunk 2]
```diff
@@ -21,19 +20,21 @@ export default class SsvMigrationService {
...
   constructor() {
     this.userDataPath = (electron.app || electron.remote.app).getPath('userData');
     this.logger = new Log(SsvMigrationService.name);
     this.ssvKeysService = new SsvKeysService();
+    this.ssvApiService = new SsvApiService();
   }
 
   async init(): Promise<void> {
     // hardcode for now
     const operatorIds = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
```
Comment (meshin-dev) [inline app/backend/services/ssv-migration/ssv-migration.service.ts:34]: Here we already can call method: https://github.com/bloxapp/ssv_api/pull/63 (@vadiminc review it please)

**Review by meshin-dev — APPROVED** [review]

File: app/backend/services/ssv-api/ssvApiService.ts (hunk 1 above)
Comment (axelrod-blox) [reply app/backend/services/ssv-api/ssvApiService.ts]: done

File: app/backend/services/ssv-migration/ssv-migration.service.ts (hunk 2 above)
Comment (axelrod-blox) [reply app/backend/services/ssv-migration/ssv-migration.service.ts:35]: done

**Pull request merged**